governance/votes/Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,174 @@
+- "Reasoning" links to the MakerDAO Forum post providing the rationale and additional context for each item. 
+- "Authorizing Parties" may also be listed (with or without links). These may include the responsible Scope Facilitator and/or Scope Advisory Council members.
+
+### Stability-Scope Defined Parameter Changes
+
+*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope)\
+*Reasoning*: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/1) "Stability Scope Parameter Changes"\
+*Authorizing Party:* [Stability Scope Facilitator](http://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/4)
+
+The Scope-defined parameter changes below have been proposed by Stability Scope Advisory Council Member [Block Analitica (BA Labs)](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-ba-labs/20813): 
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Changes
+
+- ETH-A Stability Fee will be increased by 0.12% from 3.58% to **3.70%**.
+- ETH-B Stability Fee will be increased by 0.12% from 4.08% to **4.20%**.
+- ETH-C Stability Fee will be increased by 0.12% from 3.33% to **3.45%**.
+
+#### PSM-USDP-A DC-IAM [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Change
+
+- Maximum Debt Ceiling will be increased from 0 (zero) to **120 million DAI**.
+
+### Spark Protocol Dai Direct Deposit Module (D3M) Parameter Changes
+
+*Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmQnUhZt#poll-detail)\
+*Reasoning*: [Forum post](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801) "Upcoming Spell Proposed Changes"\
+*Authorizing Parties:* [Stability Scope Facilitator](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/4); [BA Labs](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/5)
+
+Phoenix Labs has proposed increasing the Spark Protocol D3M [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) parameter as SparkLend's DAI borrow market is maxed out and requires capacity for more users. Phoenix Labs has also proposed increasing the D3M [Ceiling Increase Cooldown](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) parameter to reduce exposure to Maker Core should a time-sensitive shutdown be required. If this executive proposal passes, the following actions will occur:
+
+- The Spark Protocol D3M (DIRECT-SPARK-DAI) Maximum Debt Ceiling will be increased from 200 million DAI to **400 million DAI.**
+- The Spark Protocol D3M (DIRECT-SPARK-DAI) Ceiling Increase Cooldown will be increased from 8 hours to **12 hours.**
```

Comment:
I like to include the delta because sometimes the coded value will be a delta, so it provides all the values necessary for voters when they are reviewing

```suggestion
- The Spark Protocol D3M (DIRECT-SPARK-DAI) Maximum Debt Ceiling will be increased by 200 million DAI from 200 million DAI to **400 million DAI.**
- The Spark Protocol D3M (DIRECT-SPARK-DAI) Ceiling Increase Cooldown will be increased by 4 hours from 8 hours to **12 hours.**
```

Reply:
you've also included deltas in the scope-defined section so should aim for consistency

Reply:
Hi Patrick, I included the deltas for SF changes because these are more "customer facing" per LFW. :)